README.md

```diff
@@ -22,7 +22,7 @@ Pynecone requires the following to get started:
 -   [Node.js 16.6.0+](https://nodejs.org/en/) (Don't worry, you won’t have to write any JavaScript!)
 
 ```
```

Comment:
Might add `sh` after the backticks as a hint to readers and the highlighter that this is shell snippet.